lesson2/src/linked_list/queue.js

```diff
@@ -6,18 +6,30 @@ let Stack = require("../../../lesson1/src/stacks_queues/stack");
 class StackQueue {
   constructor() {
     // create 2 stacks here... you are not allowed to use an array or list
+    this.stack1 = [];
```

Comment:
since we imported/required Stack from lesson 1 you can do
this.stack = new Stack();
i think the rest of the exercise should work since you are only using stack methods on the array. 👍